example/zping/README.md

```diff
@@ -55,33 +55,33 @@ Linux:
    edge-router(s) and the edge-routers access to the service.
    
 
-5. Create an AppWAN and enter @ziti-ping in the service attributes and #ping in the “Endpoint Attributes”
+7. Create an AppWAN and enter @ziti-ping in the service attributes and #ping in the “Endpoint Attributes”
```

Comment:
fyi i am pretty sure you could have used just 1. over and over: 
```
1. first thing
1. second thing
```

Reply:
So you are saying do not number 1-12 in README?  Not sure I understand the ask

Reply:
you can leave it - it's fine. just next time you do this you can just use all 1.'s and markdown will render the number for you. no changes needed here, no ask. that way when you insert or add a row you don't have to renumber them yourself

Reply:
ok thanks for the tip.  I did not know that